server/src-lib/Hasura/Server/Compression.hs

```diff
@@ -24,6 +24,14 @@ compressionTypeToTxt :: CompressionType -> T.Text
 compressionTypeToTxt CTGZip   = "gzip"
 compressionTypeToTxt CTBrotli = "brotli"
 
+brotliCompressionParams :: BR.CompressParams
+brotliCompressionParams =
+  BR.defaultCompressParams{BR.compressLevel = BR.CompressionLevel4}
+
+gzipCompressionParams :: GZ.CompressParams
+gzipCompressionParams =
+  GZ.defaultCompressParams{GZ.compressLevel = GZ.compressionLevel 1}
```

Comment:
I’d just move these into local bindings (either via `let` or `where`) inside `compressResponse`, since they aren’t used anywhere else, a la https://github.com/hasura/graphql-engine/pull/2175#issuecomment-514081273.